kubeflow/trainer/algorithms_test.py

```
@@ -0,0 +1,648 @@
+
+    for algorithm_name in ALGORITHMS:
+        print("  Checking algorithm:", algorithm_name)
+        metadata = get_algorithm_pod_metadata(algorithm_name)
+
+        # Verify metadata can be repr'd (for embedding in generated code)
+        metadata_repr = repr(metadata)
+        assert isinstance(metadata_repr, str)
+        assert len(metadata_repr) > 0
+
+        # Verify metadata can be eval'd back (sanity check)
+        metadata_evaled = eval(metadata_repr)
```

Ruff (0.14.14)

[error] Use of possibly insecure function; consider using `ast.literal_eval`

(S307)